tools/devops/automation/templates/windows/build.yml

```diff
@@ -17,6 +17,17 @@ steps:
     gci env: | format-table -autosize -wrap
   displayName: 'Dump Environment'
 
+- bash: |
+    set -ex
+    ssh builder@MAC_AGENT_IP 'ls -la'
+  displayName: Diagnostics with bash
+  continueOnError: true
+
+- pws: |
+    & ssh builder@MAC_AGENT_IP 'ls -la'
+  displayName: Diagnostics with pwsh
+  continueOnError: true
```

Comment:
You probably want to use the environment variable: $Env:MAC_AGENT_IP or event do $(MAC_AGENT_IP) I suspect that is why is not working as expected.

Reply:
There are more than one typo here, but the original problem was that azure devops was complaining it couldn't find bash...